src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java

```diff
@@ -257,6 +257,7 @@ protected Uni<Optional<TableMetadata>> getSchema(
               .getKeyspaces()
               .get(CqlIdentifier.fromInternal(namespace));
     } catch (Exception e) {
+      // TODO: this ^^ is a very wide error catch, confirm what it should actually be catcing
```

Comment:
I think this is probably fine as it's just needed for plumbing Exception into async failure (instead of blowing up pipeline)